content/renderer/media/media_factory.cc

```diff
@@ -851,6 +855,10 @@ media::CdmFactory* MediaFactory::GetCdmFactory() {
   if (cdm_factory_)
     return cdm_factory_.get();
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  cdm_factory_ = std::make_unique<media::StarboardCdmFactory>();
+#else // BUILDFLAG(USE_STARBOARD_MEDIA)
```

Comment:
Should this be `else` case as the `#if BUILDFLAG(IS_FUCHSIA)` below?

Reply:
it could be, but i think that might make rebase harder if any of that code changes. With this approach, it stays as one chunk. wdyt?

Reply:
I saw `#include "media/cdm/starboard/starboard_cdm_factory.h"` above make `IS_FUCHSIA` as else cases, should we make all of them in the same way? Either staying as one chunk or make others as else cases.

Reply:
Or maybe we should put a comment as TODO here, so when we rebase or upstream the code, `BUILDFLAG(USE_STARBOARD_MEDIA)` should be one of the `else if` case.

Reply:
Wouldn't it make more sense to add it directly into the existing conditional as another elif? It would still be purely additive and would have the added benefit of the lines all being grouped together

Reply:
done.